integration-tests/environments/electron/README.md

```diff
@@ -0,0 +1,31 @@
+# Realm JS tests running in an Electron environment
+
+Currently this directory consists of:
+- An electron app in `app` where:
+  - `main.js` starts a hidden `BrowserWindow` (loading `index.html`), either runs the tests or lets the render process
+    do the heavy lifting.
+  - `index.html` simply requires `renderer.js`.
+  - `renderer.js` detects if it's supposed to run the tests and runs them.
+  - `mocha.js` creates a mocha remote client and sets globals for the tests to use.
+- `runner.js` which start the Mocha remote server and the Electron app (using the distribution package when available).
+
+When running `npm install` a post-install script will package the test suite and install that as well as Realm from an
+archive. Make sure the integration-tests folder contains a `realm-*.tgz` file before running this.
+
+## Running the tests
```

Comment:
Maybe it is because I'm not familiar with the Electron environment, but why do we need to run both the tests in the main and renderer process?

Reply:
Because they are a bit different in what libraries are linked to them. The renderer process has access to the DOM API and behaves more like a chromium browser than a Node.js process. We have had issues with SSL not loading correctly on one of them, where it was loading correctly on the other.